src/amo/reducers/versions.js

```diff
@@ -106,7 +128,19 @@ export const getVersionsBySlug = ({
   invariant(state, 'state is required');
 
   const infoForSlug = state.bySlug[slug];
-  return (infoForSlug && infoForSlug.versions) || null;
+  const versionIds = infoForSlug && infoForSlug.versionIds;
+  if (versionIds) {
+    return versionIds.map((versionId) => {
+      const version = getVersionById({ id: versionId, state });
+      if (!version) {
+        throw new Error(
+          `No version was found for slug ${slug} using versionId ${versionId}`,
```

Comment:
The coverage report says this doesn't have a test

Reply:
There doesn't actually seem to be a way to create this condition, other than manually monkeying around with the state in the test. That's generally frowned upon, but I don't think we can create a state where this error would occur just by calling reducer methods. I'll add a test to get the coverage, but I'm just trying to preempt the comment about the test.